lib/standard/creates_config_store/sets_target_ruby_version.rb

```diff
@@ -2,20 +2,27 @@ class Standard::CreatesConfigStore
   class SetsTargetRubyVersion
     def call(options_config, standard_config)
       options_config["AllCops"]["TargetRubyVersion"] = normalize_version(
-        max_rubocop_supported_version(standard_config[:ruby_version])
+        min_target_ruby_version_supported(standard_config[:ruby_version])
       )
     end
 
     private
 
-    def max_rubocop_supported_version(desired_version)
-      return desired_version unless Gem::Version.correct?(desired_version)
+    def min_target_ruby_version_supported(desired_target_ruby_version)
+      return desired_target_ruby_version unless Gem::Version.correct?(desired_target_ruby_version)
 
-      rubocop_supported_version = Gem::Version.new("2.5")
-      if desired_version < rubocop_supported_version
-        rubocop_supported_version
+      # This is minimum version that Rubocop can parse, not the minimum
+      # version it can run on (e.g. TargetRubyVersion).  See the following
+      # for more details:
+      #
+      # https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version
+      #
+      # https://github.com/rubocop/rubocop/blob/master/lib/rubocop/target_ruby.rb
+      min_target_ruby_version = Gem::Version.new("2.0")
```

Comment:
Assuming this is a magical number -- is there a better place for it to live in the repo for when we need to update it?

Reply:
I'll find a better place for this magic number.

Reply:
Added as a constant in the `sets_target_ruby_version` class.  If you have a better place for it please let me know.